include/z64.h

```diff
@@ -1653,11 +1653,22 @@ typedef struct {
 } JpegDecoderState; // size = 0x14
 
 typedef struct {
-    /* 0x0000 */ OSViMode viMode;
-    /* 0x0050 */ char unk_50[0x30];
+    /* 0x0000 */ OSViMode customViMode;
+    /* 0x0050 */ s32 viHeight;
+    /* 0x0054 */ s32 viWidth;
+    /* 0x0058 */ s32 unk58; // Right adjustment?
+    /* 0x005c */ s32 unk5C; // Left adjustment?
+    /* 0x0060 */ s32 unk60; // Bottom adjustment?
+    /* 0x0064 */ s32 unk64; // Top adjustment?
+    /* 0x0068 */ s32 viModeBase; // enum: {0: None, 1, 2, 3: NTSC}
+    /* 0x006c */ s32 viTvType;
+    /* 0x0070 */ u32 unkFlagTop; // bool
+    /* 0x0074 */ u32 unkFlagBottom; // bool
+    /* 0x0078 */ u32 unkFlagLeft; // bool
+    /* 0x007c */ u32 unkFlagRight; // bool
     /* 0x0080 */ u32 viFeatures;
-    /* 0x0084 */ char unk_84[4];
-} unk_80166528;
+    /* 0x0084 */ u32 _unused;
```

Comment:
You should leave it as an `unk` unless you are sure it is not used anywhere in the game's code. Also we don't use the `_` prefix for variables.
```suggestion
    /* 0x0084 */ char unk_84[4];
```

Reply:
Makes sense, I can put it back (as far as I could tell, it was unused)